src/layers/TrajservLayer.js

```diff
@@ -22,8 +22,14 @@ const OPERATOR_FILTER = 'operator';
 /**
  * Responsible for loading tracker data from Trajserv.
  * @class
- * @param {Object} [options]
  * @inheritDoc
+ * @param {Object} [options] Layer options.
```

Comment:
@edellucien I couldn't find the usage of options.showVehicleTraj. If it is still used, we should document this option.

Reply:
I don't think it is  used